File: BaekJoon_C/1145/1145/1145.c
#define _CRT_SECURE_NO_WARNINGS
#include<stdio.h>
#include<string.h>

int main()
{
    int count = 0;
    int* arr = (int*)malloc(sizeof(int) * 5);
    for (int i = 0; i < 5; i++) scanf("%d", &arr[i]);

    int Min = 1;

    while (1) {
        int cnt = 0;
        for (int i = 0; i < 5; i++)
        {
            if (Min >= arr[i] && Min % arr[i] == 0) cnt++;
        }

        if (cnt >= 3) break;

        Min++;
    }

    printf("%d", Min);

	return 0;
}

File: BaekJoon_python/1000~9999/1000~1999/1145.py
arr = list(map(int, input().split()))

m = 1

while 1 : 
    cnt = 0
    
    for i in range(5) : 
        if m >= arr[i] and m % arr[i] == 0 : cnt += 1
    if cnt >= 3 : break
    
    m += 1
    
print(m)